src/Components/FeelingsLog/FeelingsLog.js

```diff
@@ -1,10 +1,45 @@
-import React from "react";
+import React from 'react';
+import './FeelingsLog.css';
+import { DateTime } from 'luxon'
 
-const FeelingsLog = () => {
+const FeelingsLog = ({logs}) => {
+
+  const toTitleCase = (item) => {
+    return `${ item.slice(0, 1).toUpperCase() }${ item.slice(1) }`;
+  }
+
+  const sorted = logs
+    .map(log => ({
+      ...log,
+      date: DateTime.fromFormat(log.date, 'yyyy-MM-dd'),
```

Comment:
BIG BRAIN ENERGY! Loving the spread operator here to map while changing one property!

Reply:
agree!! I have a clarification question as well, there's that second set of parentheses inside the body of the map's callback. I played around with it and I see that it is definitely necessary, I'm just not sure I've seen that syntax before. Is there something I'm forgetting or missing about React syntax that makes that necessary?

Reply:
i dont _quite_ understant how it works, other than with *just* curlies, react thinks I am just opening a function body and its like "what do you want me to do here...? I see it as similar to when we return jsx we wrap in parens (even tho those arent necessary). I don't totally understand how it works though

Reply:
Sweet, I'll add extra parens to my toolbelt for when React is angry and I'm not sure why. Appreciate you!